catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/common/AcftbAttribute.java

```diff
@@ -29,102 +30,118 @@
 
 public class AcftbAttribute extends NitfAttributeImpl<Tre> {
 
-    private static final List<NitfAttribute<Tre>> ATTRIBUTES = new LinkedList<>();
+    private static final String PREFIX = ExtNitfUtility.EXT_NITF_PREFIX + "acftb.";
+
+    public static final String AIRCRAFT_MISSION_ID = PREFIX + "aircraft-mission-id";
+
+    public static final String AIRCRAFT_TAIL_NUMBER = PREFIX + "aircraft-tail-number";
+
+    public static final String SENSOR_ID_TYPE = PREFIX + "sensor-id-type";
+
+    public static final String SENSOR_ID = PREFIX + "sensor-id";
+
+    public static final String AIRCRAFT_TAKEOFF = PREFIX + "aircraft-take-off";
```

Comment:
In the name you have `TAKEOFF` as one word, but then the value is hyphenated (`take-off`).
Should probably be consistent here.